initfiles/bin/init_roxie

```diff
@@ -40,9 +40,6 @@ fi
 export LIBC_FATAL_STDERR_=1
 export restarts=0
 
-ulimit -c unlimited
-ulimit -n $NUM_ROXIE_HANDLES
```

Comment:
Have we done the same to dafilesrv too - used to be configurable 

Seems to me (unless I have misunderstood) that we can keep the (good) change that puts common default values code into hpcc-common, but leave the code in dalfilesrv and roxie init that overrides the ulimit -n as set by the user.